exporter/stackdriver/stackdriver.go

```diff
@@ -33,6 +33,7 @@ import (
 )
 
 // Options contains options for configuring the exporter.
+// Deprecated: This package has been moved to: contrib.go.opencensus.io/exporter/stackdriver.
```

Comment:
Add a line in between so the deprecation notice starts at its own paragraph. 

// Options contains options for configuring the exporter.
//
// Deprecated: This package has been moved to: contrib.go.opencensus.io/exporter/stackdriver.